### PR TITLE
Adding AIT*, BIT* and ABIT* to listed OMPL planners.

### PR DESCRIPTION
**Pull request by alaflaquiere**
### Description

Adding AIT*, BIT* and ABIT* to list of OMPL planners.
This PR is related to the [moveit PR#3347](https://github.com/ros-planning/moveit/pull/3347) and should only get merged after PR#3347 is merged.

### CODE REVIEW

File: doc/ompl_interface/ompl_interface_tutorial.rst [hunk 1]
```diff
@@ -65,6 +65,9 @@ Several planners that are part of the OMPL planning library are capable of optim
 * SPARS
 * SPARS2
 * Transition-based RRT (T-RRT)
+* AIT*
+* BIT*
```
Comment (yurirocha15) [inline doc/ompl_interface/ompl_interface_tutorial.rst:69]: A few lines below, it is saying that BIT* is not implemented:
```
* Batch Informed Trees (BIT*)
```
I think that line should be removed in this PR.

File: doc/ompl_interface/ompl_interface_tutorial.rst [hunk 2]
```diff
@@ -65,6 +65,9 @@ Several planners that are part of the OMPL planning library are capable of optim
...
+* ABIT*
```
Comment (yurirocha15) [inline doc/ompl_interface/ompl_interface_tutorial.rst:70]: Also, I am not sure what is the common standard here (some planners have only the abbreviation, while others have the full name as well), but I personally think writing the full name would be better.

```suggestion
* Batch Informed Trees (BIT*)
* Advanced BIT* (ABIT*)
* Adaptively Informed Trees (AIT*)
```

File: doc/ompl_interface/ompl_interface_tutorial.rst (hunk 1 above)
Comment (alaflaquiere) [reply doc/ompl_interface/ompl_interface_tutorial.rst:69]: Absolutely. I missed that.

File: doc/ompl_interface/ompl_interface_tutorial.rst (hunk 2 above)
Comment (alaflaquiere) [reply doc/ompl_interface/ompl_interface_tutorial.rst]: Got it.

**Review by rhaschke — APPROVED** [review]

**Pull request merged**